internal/dataplane/sendconfig/kong.go

```diff
@@ -37,4 +39,8 @@ type Config struct {
 	// FallbackConfiguration indicates whether to generate fallback configuration in the case of entity
 	// errors returned by the Kong Admin API.
 	FallbackConfiguration bool
+
+	// CurrentSnapshotHash stores the hash of the current snapshot. It's used to determine configuration
+	// changes, it's used for the fallback configuration generation.
+	CurrentSnapshotHash store.SnapshotHash
```

Comment:
Nit: not sure if we need the `Current` prefix. IMHO it's implied that if we'd use `SnapshotHash` that would describe the current one. WDYT?